docs/pages/server-access/guides/openssh.mdx

```diff
@@ -117,11 +146,44 @@ host.
 
 ### Issue a host certificate
 
-On your local machine, assign the IP address or fully qualified domain name of
-your Node to an environment variable.
+When creating host certificates, it is important to specify all the domain names
+and addresses that refer to your Node. If you try to connect to a Node with a 
+name or address that was not specified when creating it's host certificate,
+Teleport will reject the SSH connection.
+
+#### Finding your Node's UUID
+
+When you created a node resource earlier, a unique UUID was generated for that
+node. This is so multiple nodes with the same hostname can be uniquely identified.
```

Comment:
Can be uniquely identified by what? We should use active voice here so this is clearer ("x uses the UUID to uniquely identify multiple nodes with the same hostname" or something like that).

Reply:
Done